src/Schema/Events/Tap.ts

```diff
@@ -529,7 +529,7 @@ export interface TappedShowMore {
 /**
  * A user taps a navigation tab on iOS
  *
- * This schema describes events sent to Segment from [[tappedNavigationTab]]
+ * This schema describes events sent to Segment from [[TappedNavigationTab]]
```

Comment:
We want these links to be lowercase, so that they cross-link to the relevant helper in the docs: https://cohesion.artsy.net/interfaces/_schema_events_click_.clickedarticlegroup.html

Reply:
ah, i think i have several to edit then :)